library/oneview_version_facts.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2016-2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+ANSIBLE_METADATA = {'status': ['preview'],
```

Comment:
As mentioned on the https://github.com/HewlettPackard/oneview-ansible/pull/301#discussion_r147687457 , this `ANSIBLE_METADATA` would be better if alphabetically ordered:
```python
ANSIBLE_METADATA = {'metadata_version': '1.1',
                    'status': ['preview'],
                    'supported_by': 'community'}
```

Many of the modules are not like that currently, but it is one of the things that often get requested when sending the modules to Ansible Core, and it is considered a best practice in alphabetical order.

I'd recommend always striving for those best practices.